node/rpc/src/auth_api.rs

```diff
@@ -35,7 +38,9 @@ where
     B: Beacon + Send + Sync + 'static,
 {
     let ks = data.keystore.read().await;
-    let (token,) = params;
+    let (header_raw,) = params;
+    let token = header_raw.trim_start_matches("Bearer ");
```

Comment:
If we add case insensitivity here for the `Bearer ` prefix, we might save a stranger some time. 

`trim_start_matches` accepts a `&[char]` as a `Pattern`:
https://doc.rust-lang.org/std/primitive.str.html#method.trim_start_matches

Reply:
but doesn't accepts a `&[str]`, so that's less helpful, It might require two calls

Reply:
This approach is compatible with Lotus, so that's all I'm concerned about right now